### ISSUE TITLE
Typos and extra blank line [3 corrections needed] CHAT ReadingCenter manual 8.6.1.3 EEG arousal

### ISSUE THREAD

**Issue by JLArnold** [nsrr/chat-documentation#39]
https://www.sleepdata.org/datasets/chat/pages/manuals/polysomnography-reading-center/6-08-01-scoring-sleep-stages-and-arousals.md

```
CHAT PSG Reading Center manual 6.8.1.3 EEG arousal, Arousal Length – Wake versus Sleeps:*  typo delete the letter “s” at the end of “Sleeps”, delete the extra space [blank line] between lines 2 and 3 in the text, and add the missing "()" around the * which indicates a local scoring rule "(*)".  "Sleeps:*" should be "Sleep:(*)":
```

![image](https://cloud.githubusercontent.com/assets/16805718/12588942/2a355fb2-c429-11e5-9fa2-1a625158d7e1.png)

**Comment by michellereid** [comment]
Can you just clarify more as to what the 3 corrections are? I see one as 'wake versus sleep' but what else?

**Comment by JLArnold** [comment]
Michelle: the first correction was delete the letter "s" at the end of the word "Sleeps", the second correction was to put "( )" around the "star" after "Wake versus Sleep: (*)" the (*) is used to indicate a local scoring rule, and the third correction was to delete the "blank line" between the second and third line of text, in the original MOP this break was between two pages.
Jean

**Comment by JLArnold** [comment]
Michelle
I entered a comment on NSRR in response to this question.  I did not check all the emails from GetHub as you were doing the corrections to the documentation. I hope that you saw the three corrections:
Delete the letter s from the end of the word Sleeps
Enclose the * with “( )”,  “(*)” which indicates a local scoring rule
Third correction was to remove the blank line in the text, there may have been a page break character in the original document, the blank line is where there was a break in the text from one page to the next page.
Thanks
Jean

[cid:image001.jpg@01D15FF6.6E370C70]
